pkgs/by-name/id/ida-free/package.nix

```diff
@@ -16,51 +15,24 @@
   libsForQt5,
   libunwind,
   libxkbcommon,
-  makeDesktopItem,
   makeWrapper,
   openssl,
   stdenv,
   xorg,
   zlib,
 }:
 
-let
-  srcs = builtins.fromJSON (builtins.readFile ./srcs.json);
-in
 stdenv.mkDerivation rec {
   pname = "ida-free";
-  version = "8.4.240320";
+  version = "9.0sp1";
 
   src = fetchurl {
-    inherit (srcs.${stdenv.system} or (throw "Unsupported system: ${stdenv.hostPlatform.system}"))
-      urls
-      sha256
-      ;
+    url = "https://archive.org/download/ida-free-pc_90sp1_x64linux/ida-free-pc_90sp1_x64linux.run";
```

Comment:
This breaks the package on *-darwin. Why not adjust `srcs.json` with the new installers?